frontend/pages/create-edit-user.js

```diff
@@ -0,0 +1,424 @@
+            case 'DISCORD_INPUT':
+                return {
+                    ...state,
+                    links: {
+                        ...links,
+                        discord: {
+                            value: action.val,
+                            isValid: action.val.match(validators.discord),
+                        },
+                    },
+                };
+            case 'GITHUB_INPUT':
+                return {
+                    ...state,
+                    links: {
+                        ...links,
+                        github: {
+                            value: action.val,
+                            isValid: action.val.match(validators.github),
+                        },
+                    },
+                };
+            case 'INPUT_BLUR':
+                return state;
+            case 'FORM_RESET':
+                return { ...defaultFormValues };
+        }
+    };
+
+export default function UserTest(props) {
+    const [nameRef, emailRef, usernameRef, websiteRef, bioRef] = [
+            useRef(),
+            useRef(),
+            useRef(),
+            useRef(),
+            useRef(),
+        ],
+        links = [
+            { name: 'Twitter', ref: useRef(), type: 'url' },
+            { name: 'Lenster', ref: useRef(), type: 'url' },
+            { name: 'Instagram', ref: useRef(), type: 'url' },
+            { name: 'Discord', ref: useRef(), type: 'text' },
+            { name: 'GitHub', ref: useRef(), type: 'url' },
+        ],
+        [userFormState, dispatchUserForm] = useReducer(userFormReducer, {
+            ...defaultFormValues,
+        }),
+        [hasIPFonts, setHasIPFonts] = useState(false),
+        [lensHandle, setLensHandle] = useState(''),
+        [txStatus, setTxStatus] = useState('DEFAULT'),
+        formInputChangeHandler = e => {
+            let rawID = e.target.id.replace('user-', '');
+            dispatchUserForm({
+                type: `${rawID.toUpperCase()}_INPUT`,
+                val: e.target.value,
+            });
+        },
+        validateFormInputHandler = () => {
+            dispatchUserForm({ type: 'INPUT_BLUR' });
+        },
+        resetForm = () => {
+            dispatchUserForm({ type: 'FORM_RESET' });
+            setTxStatus('DEFAULT');
+        };
+
+    // Check if the user has IPFonts and lens profile
+    useEffect(() => {
+        (async () => {
+            try {
+                // Has IPFonts profile
+                const ipfontsContract = await connectContract();
+                if (!ipfontsContract) {
+                    alert('Cound not connect to contract');
+                    return;
+                }
+                const ipfontsProfile = await ipfontsContract.addressToUser(
+                    props.address
+                );
+                setHasIPFonts(Boolean(ipfontsProfile.createdAt.toNumber()));
+                // Has Lens profile
+                const getLensUser = await lensClient.mutate({
```

Comment:
This should be a query...
```suggestion
                const getLensUser = await lensClient.query({
```

Reply:
How can I missed that! 😅

Reply:
Now is a [query](https://github.com/interplanetaryfonts/interplanetaryfonts.xyz/blob/09f178c3576c21dbcd3dc2e598a852abefeb02db/frontend/pages/create-edit-user.js#L94-L97) and actually I ended using the getDefaultProfile we had on the api